neoforge/src/main/java/org/mcaccess/minecraftaccess/neoforge/MainClassNeoForge.java

```diff
@@ -17,10 +18,8 @@ public class MainClassNeoForge {
     public MainClassNeoForge(IEventBus modEventBus, ModContainer container) {
         modEventBus.register(this);
         container.registerExtensionPoint(IConfigScreenFactory.class, (modContainer, parent) -> AutoConfig.getConfigScreen(Config.class, parent).get());
-    }
-
-    @SubscribeEvent
-    private void onClientSetup(final FMLClientSetupEvent event) {
-        MainClass.init();
+        final NeoForgeLoadContext context = new NeoForgeLoadContext(modEventBus);
```

Comment:
We do not currently use immutable local variables anywhere and this variable is not special. If you want to adopt that as a standard it should be subject to proper discussion and a refactor to apply it everywhere instead of just one random variable somewhere.